Proxy/proxy.sample.ini

```diff
@@ -6,9 +6,16 @@ PORT=8000
 FILENAME=log.db
 SCHEMANAME=db.sql
 
+[TESTDATABASE]
+FILENAME=log.test.db
+
 [PROXY]
 LOG=0
 UNITS=1
```

Comment:
Can run with that.  Multiple test units needs quite a bit more thought.  I'm slightly confused about why the port is written to the DB and not the callsign or nodeid.  At some later point in time, after a configuration has been changed for example, the log no longer makes sense.  Also, I could imagine that the DB log could be interspersed with packets from multiple nodeids and the only way to differentiate is currently port number.  If the callsign and nodeid were logged a single DB could play back traffic for a number of units.  I'm not sure about the design and why it is currently so that may not be what's desired.